governance/polls/Ratification Poll - MIP4c3-SP1 - March 14, 2022.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Ratification Poll for MIP64 Amendments (MIP4c3-SP1) - March 14, 2022
+summary: Signal your support or opposition to amending MIP64 to clarify critical web bounty amount.
+discussion_link: https://forum.makerdao.com/t/signal-request-promote-mip4c3-sp1-amend-mip64-to-clarify-critical-web-bounty-amount-to-on-chain-poll/13649
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-03-14T16:00:00
+end_date: 2022-03-28T16:00:00
```

Comment:
Should duration be 3 days because MIP4c3 uses the weekly cycle and not the monthly cycle? Also not sure how this works with minimum participation values etc. @blimpa can you confirm?

Reply:
Yes, this one should be three days

Unsure about the minimum participation values :/